apps/2i/hotel/frontend/src/app/admin/_Components/HotelAmenitiesDialog.tsx

```
@@ -0,0 +1,38 @@
+'use client';
+
+import * as React from 'react';
+import { Dialog, DialogContent, DialogHeader, DialogTitle, DialogFooter, DialogTrigger } from '@/components/ui/dialog';
+import { Button } from '@/components/ui/button';
+
+export const LocationDialog = () => {
+  const handleSave = () => {};
```

ESLint

[error] Unexpected empty arrow function.

(@typescript-eslint/no-empty-function)